integration/otlp_ingestion_test.go

```diff
@@ -72,4 +74,18 @@ func TestOTLPIngestion(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, model.ValMatrix, rangeResult.Type())
 	require.Equal(t, expectedMatrix, rangeResult.(model.Matrix))
+
+	// Push series with histograms to Mimir
+	series = []prompb.TimeSeries{
+		{
+			Labels: []prompb.Label{
+				{Name: "foo", Value: "bar"},
+			},
+			Histograms: []prompb.Histogram{
+				remote.HistogramToHistogramProto(1337, tsdb.GenerateTestHistograms(1)[0]),
+			},
+		},
+	}
+	_, err = c.PushOTLP(series)
```

Comment:
We also need to query after push to verify that the histograms have been ingested (like how we are doing above).

Reply:
I've missed that the support for native histogram was just added https://github.com/grafana/mimir/commit/c1cfa0367dc87440546dabc9081ced3276d3eb77. I'll update that.

Reply:
Oh, right. `main` branch does not have support for querying yet, that commit is only for ingestion. I think a TODO saying test for querying should be enough right now.

Reply:
Sure, just added the TODO note.